### PR TITLE
chore: improve about panel and sidenavs

### PR DESCRIPTION
**Pull request by acalinica**
## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx [hunk 1]
```diff
@@ -38,27 +38,32 @@ export const OrganizationSidenav = () => {
...
-      <Flex gap='2' align='center' mb='4'>
-        <IconButton
-          size='md'
-          variant='ghost'
-          aria-label='Go back'
-          onClick={() => router.push('/organization')}
-          icon={<Icons.ArrowNarrowLeft color='gray.700' boxSize='6' />}
-        />
+      <Tooltip
+        label='Organization Name Lorem ipsum sin dolor amit sumit'
```
Comment (Kasia9090) [inline packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx]: Lorem ipsum :)

File: packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx [hunk 2]
```diff
@@ -38,27 +38,32 @@ export const OrganizationSidenav = () => {
...
-        </Text>
-      </Flex>
+          <Text
+            fontSize='lg'
+            fontWeight='semibold'
+            color='gray.700'
+            noOfLines={1}
+            wordBreak='keep-all'
+          >
+            Organizationabcdefg
```
Comment (Kasia9090) [inline packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx]: copy :)

**Review by Kasia9090 — CHANGES_REQUESTED** [review]

File: packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx (hunk 1 above)
Comment (acalinica) [reply packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx]: Thing is I need the value of the Tenant which I'm not sure where to pick it from.

File: packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx (hunk 2 above)
Comment (acalinica) [reply packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx]: Same here...

File: packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx (hunk 1 above)
Comment (Kasia9090) [reply packages/apps/spaces/app/components/Sidenavs/OrganizationSidenav.tsx]: import { useTenantName } from '@spaces/hooks/useTenant';

You can use this hook

**Review by edifirut — DISMISSED** [review]

**Review by edifirut — APPROVED** [review]

**Review by Kasia9090 — APPROVED** [review]

**Pull request merged**